### ISSUE TITLE
[BUG]: Angular v18 support

### ISSUE THREAD

**Issue by kingjordan** [state-adapt/state-adapt#96]
## Description
I get an error during npm install with angular v18 
...

## Error
```
npm ERR! peer @angular/core@"^14.0.0 || ^15.0.0 || ^16.0.0 || ^17.0.0" from @state-adapt/angular@2.0.7
npm ERR! node_modules/@state-adapt/angular
npm ERR!   @state-adapt/angular@"^2.0.7" from the root project
```

**Comment by mfp22** [comment]
Okay I need to test it and add it to npm for version 18. It should work fine. I think you might be able to do `npm install --force` in the meantime

**Comment by kingjordan** [comment]
thanks mike,

I used --force to make it work and there are no issues on my end, I can see its just a version check that needs the update.  

Cheers

**Comment by mfp22** [comment]
Finally published this in Angular 2.0.8

Also worked on some scripts to make publishing faster
